containers/kitchensink-aws/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,181 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+# Used to install awsenv - https://github.com/base2Services/aws-env
+FROM base2/awsenv as awsenv
+
+# Chose one of the following depending on if you want dotnet core SDK or not
+FROM mcr.microsoft.com/dotnet/core/sdk:2.2-bionic
+#FROM buildpack-deps:bionic-scm
+
+#
+# Or your actual UID, GID on Linux if not the default 1000
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Suppress an apt-key warning about standard out not being a terminal. Use in this script is safe.
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
+
+# Configure apt and install packages
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils 2>&1 \
+# Install pre-requisite packages
+    && apt-get -y install \
+        curl \
+        git \
+        groff \
+        iproute2 \
+        less \
+        locales \
+        lsb-release \
+        procps \
+        sudo \
+        unzip \
+        wget \
+# Creata a non-root user
+    && groupadd --gid $USER_GID $USERNAME \
+    && useradd -s /bin/bash --uid $USER_UID --gid $USER_GID -m $USERNAME \
+    && echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME \
+    && chmod 0440 /etc/sudoers.d/$USERNAME \
+# Setup a locale compatible with perl/brew
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 \
+# Clean up layer
+    && apt-get autoremove -y \
+    && apt-get clean -y \
+    && rm -rf /var/lib/apt/lists/*
+
+# Ensure we have a locale so perl/brew don't complain
+ENV LANG en_US.utf8
+
+# Install awsenv
+COPY --from=awsenv /awsenv /usr/local/bin/awsenv
+
+#
+# Install awscli
+RUN apt-get update && apt-get -y install python3-pip \
+    && apt-get autoremove -y \
+    && apt-get clean -y \
+    && pip3 install awscli \
+    && rm -rf /var/lib/apt/lists/*
+#
+# Install Python3
```

Comment:
Shouldn't this go before you install python3-pip? I believe installing python3 is redundant afterwards. If is it not, why can't they both be installed at the same time in one `apt install`?